common/IElectronBridge.ts

```diff
@@ -30,6 +30,11 @@ export interface IElectronBridgeSync {
 export  interface IElectronBridgeAsync {
   openFileDialog(options: OpenDialogOptions): Promise<OpenDialogReturnValue>;
   openMessageDialog(options: MessageBoxOptions): Promise<MessageBoxReturnValue>;
+  // keytar
+  keytarGetPassword(service: string, account: string): Promise<string | null>
```

Comment:
We may end up wanting to export of amore specific interface rather than a general one. For example, if our app always sets the service to "DiceKeys", there's no need to export an API with a customizable service string.

Reply:
Updated withe the following interface
```
export interface IElectronBridgeDiceKeysStoreAsync {
  getDiceKey(id: string): Promise<string | null>
  setDiceKey(id: string, humanReadableForm: string): Promise<void>
  deleteDiceKey(id: string): Promise<boolean>
  getDiceKeys(): Promise<Array<{ id: string, humanReadableForm: string}>>
}
```